Objects/weakrefobject.c

```diff
@@ -128,19 +130,20 @@ gc_clear(PyWeakReference *self)
 
 
 static PyObject *
-weakref_call(PyWeakReference *self, PyObject *args, PyObject *kw)
+weakref_vectorcall(PyObject *callable, PyObject *const *args,
```

Comment:
At the ABI level, ``PyWeakReference*`` and ``PyObject*`` are just pointers, so you can use directly ``PyWeakReference*``, no? I don't think that clang Control Flow Integrity (CFI) cares about that, does it?